CMakeLists.txt

```diff
@@ -323,6 +323,7 @@ add_swift_library(Foundation
                     ${CoreFoundation_INTERFACE_LIBRARIES}
                     ${WORKAROUND_SR9138}
                     ${WORKAROUND_SR9995}
+                    -Xlinker;/FORCE
```

Comment:
This is wrong.  We shouldn't be using `/FORCE`.  Furthermore, this needs to be specific to Windows.  What symbol is being defined multiply?  Those should have all been fixed by now.

Reply:
I squash two latest commits.
Right now I cannot list symbols which causing LNK1169 error cos hasnot access to Win32 machine. I will try to rebuild Foundation without /FORCE flag later today and attach build-log